src/phocus_windows.go

```diff
@@ -47,12 +47,10 @@ func main() {
 }
 
 func (p *program) Init(env svc.Environment) error {
-	/*
-		if ! env.IsWindowsService() && *idgui != "yes" {
-		    failPrint("Sorry! You need to run this as a Windows service.")
-		    os.Exit(1)
-		}
-	*/
+	if !env.IsWindowsService() && !*idgui {
+		failPrint("Sorry! Don't run this binary. It's probably already running as a Windows service (CSSClient).")
```

Comment:
Won't this just print the message and exit it immediately if they double click the `exe`?

Reply:
Good point haha, I was thinking that they would run it from the command line like `./phocus`. I'll add a 5 second delay.